types/three/three-core.d.ts

```diff
@@ -5171,7 +5181,7 @@ export class Mesh extends Object3D {
     setDrawMode(drawMode: TrianglesDrawModes): void;
     updateMorphTargets(): void;
     getMorphTargetIndexByName(name: string): number;
-    raycast(raycaster: Raycaster, intersects: any): void;
+    raycast(raycaster: Raycaster, intersects: Intersect[]): Intersect[];
```

Comment:
Does `raycast` indeed return an array of intersects? I was looking through the [sourcecode](https://github.com/mrdoob/three.js/blob/5e7a7b83e1e182e662fd7b60a2674f85c9a57b35/src/objects/Mesh.js#L225) and couldn't find a return statement that returned Intersects. Only void.

I believe the THREE documentation for the `raycast` method is wrong (where is suggests it returns an array)